### ISSUE TITLE
Image stays blurred with lqip-loader

### ISSUE THREAD

**Issue by iamjordan** [cyrilwanner/next-optimized-images#138]
Hi there,

I just have an issue with using lqip-loader with this package. I have:

1) enabled the next-optimised-images plugin in next.config.js file 
2) npm install lqip-loader 
3) added "?lqip" to my img src (i.e. <img src={require('../../images/hero.png?lqip')}/> 

When I check to see the result the image is blurred, but it seems to remain that way. I have checked and it is the result for Chrome and FF. I have tried w/ other packages such as 'webp' and they work with next-optimised-images perfectly.

Any help would be greatly appreciated as I have been fiddling around with it for a few hours with no luck. 

Thanks.

1)
<img width="544" alt="1" src="https://user-images.githubusercontent.com/8903600/77644008-78d14b00-6fb4-11ea-88c1-03f1e29b00bf.png">

2)
<img width="296" alt="2" src="https://user-images.githubusercontent.com/8903600/77644019-7cfd6880-6fb4-11ea-9f60-ef14e37c93d5.png">

3)
<img width="541" alt="3" src="https://user-images.githubusercontent.com/8903600/77644031-81298600-6fb4-11ea-9c79-cca6be12a051.png">

Blurred still image.
<img width="1383" alt="4" src="https://user-images.githubusercontent.com/8903600/77644036-84bd0d00-6fb4-11ea-87ef-737603024843.png">

**Comment by AbboudAfaneh** [comment]
+1 
I'm facing the same issue, it feels like there is a step we missed maybe, a way to replace the image once the big one loaded, but the docs are not clear on that and i've searched a lot and got nothing.

**Comment by kochie** [comment]
Yes so when you use the lqip loader it will return a string that is just the base64 lqip image. You need to add the image later.

Example from a react component I made.

```typescript
async function getImage(): Promise<void> {
    return new Promise((resolve, reject) => {
      try {
        if (imgRef.current) {
          imgRef.current.onload = (): void => {
            imgRef.current?.classList.add(imageStyle.sharpen)
            resolve()
          }
          imgRef.current.src = src
          imgRef.current.onanimationend = (): void =>
            imgRef.current?.classList.remove(
              imageStyle.blur,
              imageStyle.sharpen
            )
        } else {
          console.warn('imgRef does not have a current reference!')
        }
      } catch (error) {
        console.error(error)
        reject(error)
      }
    })
  }
```

**Comment by goncalogteixeira** [comment]
Can you please give more context? I also have an issue with this. Can you give an example of the usage of the component you shared?
